### ISSUE TITLE
any future update to use the Gibbs Seawater package?

### ISSUE THREAD

**Issue by santvn** [modscripps/mADCP#1]
Hi,

The seawater package hasn't been updated recently. The Gibbs Seawater Toolbox is taking a new standard for processing CTD data (http://www.teos-10.org/pubs/gsw/html/gsw_contents.html). Is there any future plan to use it for this toolbox?

-San-

**Comment by gunnarvoet** [comment]
Good point! I'll be processing ADCP data during or after the FLEAT cruise in June 2016 and will change to the gsw toolbox if no one else will have done it by then.

**Comment by santvn** [comment]
I doubt that anyone will have it done by then.
-San-
